crates/solver/src/solver.rs

```diff
@@ -210,7 +210,7 @@ impl FromStr for SolverAccountArg {
     }
 }
 
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
For some reason, the `value_parser` conversion inference mechanism in `clap` requires this bound.